This patch temporarily comments out code from openwrt generic patch 120-bridge_allow_receiption_on_disabled_port.patch.
Its unclear at the moment why the packet should be sent up when bridge port is down. This is causing race condition
on system boot up and __nf_conntrack_confirm crash is observed. Only when root cause is found for crash, we can think of removing 
this patch.

diff --git a/net/bridge/br_input.c b/net/bridge/br_input.c
--- a/net/bridge/br_input.c
+++ b/net/bridge/br_input.c
@@ -220,6 +220,7 @@ rx_handler_result_t br_handle_frame(stru
 
 forward:
 	switch (p->state) {
+#if 0
 	case BR_STATE_DISABLED:
 		if (ether_addr_equal(p->br->dev->dev_addr, dest))
 			skb->pkt_type = PACKET_HOST;
@@ -231,7 +232,7 @@ forward:
 		BR_INPUT_SKB_CB(skb)->brdev = p->br->dev;
 		br_pass_frame_up(skb);
 		break;
-
+#endif
 	case BR_STATE_FORWARDING:
 		rhook = rcu_dereference(br_should_route_hook);
 		if (rhook) {
